concent_api/utils/api_view.py

```diff
@@ -81,6 +82,7 @@ def wrapper(request, *args, **kwargs):
             )
             return JsonResponse({'error': str(exception)}, status = 400)
         if isinstance(response_from_view, Message):
+            assert response_from_view.sig != _fake_sign(None)
```

Comment:
This should be changed to `assert response_from_view.sig == None`. If we do need to return a signed message we should clear its signature explicitly.